pkgs/unified_analytics/lib/src/initializer.dart

```diff
@@ -84,8 +84,8 @@ class Initializer {
   }) {
     final now = sessionIdOverride ?? clock.now();
     sessionFile.createSync(recursive: true);
-    sessionFile
-        .writeAsStringSync('{"session_id": ${now.millisecondsSinceEpoch}}');
+    sessionFile.writeAsStringSync(
+        '{"session_id": ${now.millisecondsSinceEpoch}, "last_ping": null}');
```

Comment:
Explanation for why I think this is necessary:
- https://github.com/dart-lang/tools/issues/252#issuecomment-1997883030